agents/plugins/mk_oracle

```diff
@@ -1290,7 +1290,77 @@ sql_recovery_status () {
 
 
 sql_rman () {
-    if [ "$NUMERIC_ORACLE_VERSION" -ge 121 ]; then
+    if [ "$NUMERIC_ORACLE_VERSION" -ge 122 ]; then
```

Comment:
Since the queries for 12.1 and 12.2+ are almost identical, we have a lot of duplicated code now. Maybe there is a way to improve on this. What I have in mind is something like:
```
if [ "$NUMERIC_ORACLE_VERSION" -ge 121 ]; then
    local archive_dest_target_cond = "b.target = 'PRIMARY'"
    if [ "$NUMERIC_ORACLE_VERSION" -ge 122 ]; then
         archive_dest_target_cond = "b.target IN ('PRIMARY', 'LOCAL')"
    fi
    ...
    echo "select /*${HINT_RMAN} check_mk rman1 */ upper(name)
    ...
    where ${archive_dest_target_cond}
    ..."
```
See also https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html